### PR TITLE
CTX-6016: Added prompt for endpoint invocation price if near wallet i…

### PR DESCRIPTION
**Pull request by bogdant36**
…d is set.

### CODE REVIEW

File: coretex/cli/modules/node.py [hunk 1]
```diff
@@ -378,13 +389,21 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
 
             if nearWalletId != config_defaults.DEFAULT_NEAR_WALLET_ID:
                 config["nearWalletId"] = nearWalletId
+                endpointInvocationPrice = clickPrompt(
+                    "Enter the price of a single endpoint invocation",
+                    config_defaults.DEFAULT_ENDPOINT_INVOCATION_PRICE,
+                    type = float
+                )
+
```
Comment (dule1322) [inline coretex/cli/modules/node.py:397]: Delete this newline

File: coretex/cli/modules/node.py [hunk 2]
```diff
@@ -378,13 +389,21 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
 
             if nearWalletId != config_defaults.DEFAULT_NEAR_WALLET_ID:
                 config["nearWalletId"] = nearWalletId
+                endpointInvocationPrice = clickPrompt(
+                    "Enter the price of a single endpoint invocation",
+                    config_defaults.DEFAULT_ENDPOINT_INVOCATION_PRICE,
```
Comment (dule1322) [inline coretex/cli/modules/node.py:394]: What happens if the price which is entered is 0? Should that be allowed? Can you check this with backend team

File: coretex/cli/modules/node.py [hunk 3]
```diff
@@ -378,13 +389,21 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
 
             if nearWalletId != config_defaults.DEFAULT_NEAR_WALLET_ID:
                 config["nearWalletId"] = nearWalletId
+                endpointInvocationPrice = clickPrompt(
```
Comment (dule1322) [inline coretex/cli/modules/node.py:392]: Display this value in configuration preview

**Review by dule1322 — CHANGES_REQUESTED** [review]

File: coretex/cli/modules/node.py [hunk 4]
```diff
@@ -172,8 +172,15 @@ def shouldUpdate(image: str) -> bool:
...
 
-def registerNode(name: str, nodeMode: NodeMode, publicKey: Optional[bytes] = None, nearWalletId: Optional[str] = None) -> str:
-    params = {
+def registerNode(
+        name: str,
+        nodeMode: NodeMode,
+        publicKey: Optional[bytes] = None,
+        nearWalletId: Optional[str] = None,
+        endpointInvocationPrice: Optional[float] = None
+    ) -> str:
```
Comment (dule1322) [inline coretex/cli/modules/node.py:181]: Fix function definition formatting

**Comment by bogdant36** [comment]
Marked as draft until related backend task is merged.

File: coretex/cli/modules/node.py [hunk 5]
```diff
@@ -378,13 +389,24 @@ def configureNode(config: Dict[str, Any], verbose: bool) -> None:
...
                 config["nearWalletId"] = nearWalletId
+                endpointInvocationPrice = clickPrompt(
+                    "Enter the price of a single endpoint invocation",
+                    config_defaults.DEFAULT_ENDPOINT_INVOCATION_PRICE,
+                    type = float
+                )
+
+                if isinstance(endpointInvocationPrice, float):
+                    if endpointInvocationPrice < 0.0:
+                        endpointInvocationPrice = 0
```
Comment (dule1322) [inline coretex/cli/modules/node.py:400]: If a user did not enter the correct value prompt them to enter the value again

File: coretex/cli/modules/node.py [hunk 6]
```diff
@@ -255,6 +265,20 @@ def promptRam(config: Dict[str, Any], ramLimit: int) -> int:
     return nodeRam
 
 
+def promptInvocationPrice(config: Dict[str, Any], retryCount: int = 0) -> float:
```
Comment (dule1322) [inline coretex/cli/modules/node.py:268]: Should retry count be removed?

File: coretex/cli/modules/node.py [hunk 7]
```diff
@@ -255,6 +265,20 @@ def promptRam(config: Dict[str, Any], ramLimit: int) -> int:
     return nodeRam
 
 
+def promptInvocationPrice(config: Dict[str, Any]) -> float:
```
Comment (dule1322) [inline coretex/cli/modules/node.py:268]: Is config needed as a parameter?

**Review by dule1322 — APPROVED** [review]

**Pull request merged**